frontend/src/metabase/admin/datamodel/components/FormInput/FormInput.tsx

```diff
@@ -14,6 +14,7 @@ const FormInput = forwardRef(function FormInput(
   return (
     <FormInputRoot
       {...props}
+      value={props.value ?? ""}
```

Comment:
I was running into this error 
```
Warning: A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component.*
```

which I found out was because `props.value` starts off as `undefined` and then switches to an actual value, which causes the error. I've just added this to suppress that error, but need to make sure if there's any actual consequences for this